CI/start-worker.sh

```diff
@@ -27,13 +27,14 @@ cat > worker.config <<EOF
   "requiredDiskSpaceMegabytes": 512,
   "signingKeyLocation": "worker.key",
   "tasksDir": "tasks",
+  "rootURL": "https://community-tc.services.mozilla.com",
```

Comment:
mmmm that's not going to work without #231, is it?

Reply:
Is there a url that would work with the current tc instance?

Reply:
Ah, `https://taskcluster.net` should do.

Sorry for the silly mistakes -- it's a bit past my bedtime :)